server/application/application.go

```diff
@@ -68,6 +69,13 @@ func (s *Server) List(ctx context.Context, q *ApplicationQuery) (*appv1.Applicat
 
 // Create creates an application
 func (s *Server) Create(ctx context.Context, a *appv1.Application) (*appv1.Application, error) {
```

Comment:
Should we just change the RPC signature to ApplicationCreateRequest instead of Application? Maybe not for 0.4.4 but for 0.5.0? Storing into context hides the API capabilities.